Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.3
+// swift-tools-version:5.6
```

Comment:
Bumping the `swift-tools-version` created a couple of warnings in `Package.swift`. Could you resolve those?